src/app/client/src/app/plugins/profile/components/profile-page/profile-page.component.html

```diff
@@ -20,12 +20,12 @@
     </div>
     <div class="profile-user-label font-w-bold d-flex text-center relative position">{{userProfile.firstName | titlecase}} {{userProfile.lastName |
       titlecase}}
-      <div class="computer only">
+      <!-- <div class="computer only">
```

Comment:
Remove this commented item 
dont keep it commented

Reply:
@rajeevsathish  Kindly check now.